### DISCUSSION TITLE
Seltene Dachformen

### DISCUSSION THREAD

**Discussion by biketeur** [streetcomplete/StreetComplete#3562] [Q&A]
Hallo zusammen,
ich bin in einer Siedlung auf 3 Dachformen gestoßen, die ich in SC nicht finde:
versetztes Pultdach
Schmetterlingsdach
Bogendach
Zur visuellen Darstellung siehe
http://www.wikiwand.com/de/dachform
Wie würdet ihr diese Dachformen taggen?

**Comment by matkoniecz** [comment]
Note that SC follows OSM tagging, described for example at https://wiki.openstreetmap.org/wiki/Key:roof:shape

If some form is missing then it should be added there after confirmation/discussion on OSM Wiki and or tagging mailing list and/or openstreetmap forum.

![screen](https://user-images.githubusercontent.com/899988/144723170-69599bb7-5625-4117-b689-16d1fc2d9878.png)

that was a final trigger for "stop asking for roof shape by default" #3161 (though making clear how it should be tagged would not be reason to start asking for default, it is just one of many tricky roof shapes)
